File: OnlineShopAPI/AccountMicroservice/Account.API/Models/UpdateAccountRequest.cs
namespace Account.API.Models
{
    public class UpdateAccountRequest
    {
        public long AccountId { get; set; } 
        
        public string NewName { get; set; }
        
        public string NewEmail { get; set; }
        
        public string NewPassword { get; set; }
    }
}

File: OnlineShopUI/OnlineShop/src/app/model/requests/update-account-request.ts
export class UpdateAccountRequest {
    accountId: number;
    newName: string;
    newEmail: string;
    newPassword: string;

    constructor(accountId: number, newName: string, newEmail: string, newPassword: string){
        this.accountId = accountId;
        this.newName = newName;
        this.newEmail = newEmail;
        this.newPassword = newPassword;
    }
}
